tests/ParserTest.php

```diff
@@ -1180,4 +1238,28 @@ public function lonelyImport()
         $sExpected = "@import url(\"example.css\") only screen and (max-width: 600px);";
         self::assertSame($sExpected, $oDoc->render());
     }
+
+    /**
+     * @test
+     */
+    public function functionArithmeticInFile()
+    {
+        $oDoc = self::parsedStructureForFile('function-arithmetic', Settings::create()->withMultibyteSupport(true));
+        $sExpected = 'div {height: max(300,vh + 10);}
```

Comment:
AFAICT, vh without a number in front is still not valid and I’d prefer it would throw an error in strict mode.

Reply:
Indeed, it is invalid, however this seems to be outside of the scope of these changes. I guess we need a separate issue for this. Also is it a concern of the parser to validate the semantic meaning of identifiers it encounters?

Reply:
> Also is it a concern of the parser to validate the semantic meaning of identifiers it encounters?

I agree it's not.

Moving forwards, we've agreed to replace strict mode parsing with a logging mechanism.  And try to replicate what most browsers do when encountering invalid syntax.  A unit with no number in front should therefore probably be parsed as zero of that unit.  And, as a bonus, a separate change would log the lack of a number.

So, for now, we don't need to worry about strict mode, but should create issues for things that could be logged in future.